src/main/java/uk/gov/ons/ctp/response/sample/ingest/CsvIngesterBusiness.java

```diff
@@ -110,14 +111,24 @@ public SampleSummary ingest(MultipartFile file)
         samplingUnitList.add(businessSampleUnit);
 
       }
+      expectedCI = calculateExpectedCollectionInstruments(samplingUnitList);
 
       businessSurveySample.setSampleUnits(samplingUnitList);
 
-      sampleSummary = sampleService.processSampleSummary(businessSurveySample, samplingUnitList);
+      sampleSummary = sampleService.processSampleSummary(businessSurveySample, samplingUnitList, expectedCI);
 
     return sampleSummary;
   }
 
+  private Integer calculateExpectedCollectionInstruments(List<BusinessSampleUnit> samplingUnitList) {
+    //TODO: get survey classifiers from survey service, currently using formtype for all business surveys
+    List<String> formTypes = new ArrayList<>();
```

Comment:
It might be more readable if you use a set for the count as it will only allow unique values in or use streams (maybe less readable?)

```java
  private Integer calculateExpectedCollectionInstruments(List<BusinessSampleUnit> samplingUnitList) {
    //TODO: get survey classifiers from survey service, currently using formtype for all business surveys
    Set<String> formTypes = new HashSet<>();
    for (BusinessSampleUnit businessSampleUnit : samplingUnitList) {
      formTypes.add(businessSampleUnit.getFormtype2());
    }
    return formTypes.size();
  }
```

```java
  private Long calculateExpectedCollectionInstruments(List<BusinessSampleUnit> samplingUnitList) {
    //TODO: get survey classifiers from survey service, currently using formtype for all business surveys
    return samplingUnitList.stream()
            .map(BusinessSampleUnit::getFormtype2)
            .distinct().count();
  }
```